docs/track_types.rst

```diff
@@ -228,6 +231,7 @@ Options
     }
   }
 
+- **valueScaleMin/valueScaleMax**: Absolute values limiting the value to y-position scale.
```

Comment:
I am not sure I'd fully understand the description if I wasn't a core developer. How about:

Absolute values limiting the value scale, which is used to determine the y-position.

Just an idea :)

Reply:
```suggestion
- **valueScaleMin/valueScaleMax**: Absolute values limiting the limits of the value scale, which is used to determine y-position (in 1D tracks) or color (heatmap) tracks.
```